_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -219,4 +221,128 @@ function check_plan_deactivate_modules( $page ) {
 			'deactivate' => $to_deactivate
 		);
 	}
+
+	static function load_wrapper_styles( ) {
+		$rtl = is_rtl() ? '.rtl' : '';
+		wp_enqueue_style( 'dops-css', plugins_url( "_inc/build/admin.dops-style{$rtl}.css", JETPACK__PLUGIN_FILE ), array(), JETPACK__VERSION );
+		wp_enqueue_style( 'components-css', plugins_url( "_inc/build/style.min{$rtl}.css", JETPACK__PLUGIN_FILE ), array(), JETPACK__VERSION );
+		$custom_css = '
```

Comment:
Is there a particular reason for this CSS to be written here? If possible, it should be moved to a stylesheet.